CHANGELOG.md

```diff
@@ -48,6 +48,8 @@
 - `receiver/otlp`: Make logs related to gRCPC and HTTP server startup clearer (#6174)
 - Add prometheus metric prefix and constant service attributes to Collector's own telemetry when using OpenTelemetry for internal telemetry (#6223)
 - `exporter/logging`: Apply consistent rendering of map values (#6244)
+- Add support in the confmap.Resolver to expand embedded config URIs inside configuration. (#6276)
+  - For usage details see https://github.com/open-telemetry/opentelemetry-collector/pull/4742.
```

Comment:
```suggestion
  - For usage details see https://github.com/open-telemetry/opentelemetry-collector/pull/6276.
```

Reply:
Removed the line, no need since is already the PR listed.